docs/resource-policy/policy/balloons.md

```diff
@@ -679,6 +679,21 @@ instrumentation:
   metrics:
     enabled: # use '*' instead for all available metrics
     - policy
-logger:
-  Debug: policy
+log:
+  debug:
+  - policy     # to follow policy's internal logic
+  - nri-plugin # to see what the policy saw and to hear what it said (there is a lot of it)
+  source: true
+```
+
+Read metrics:
```

Comment:
Maybe a note that this only works "locally" i.e. you need direct connectivity to the pod ip(?)